sources/packages/backend/workflow/test/2023-2024/parttime-assessment/parttime-assessment-2023-2024-eligibility-CSGP.e2e-spec.ts

```diff
@@ -0,0 +1,48 @@
+import { PROGRAM_YEAR } from "../constants/program-year.constants";
+import {
+  createFakeConsolidatedPartTimeData,
+  executePartTimeAssessmentForProgramYear,
+} from "../../test-utils";
+
+describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-eligibility-CSGP.`, () => {
+  it(
+    "Should determine CSGP as eligible when total assessed need is greater than 0 " +
+      "and application PD/PPD status is true for a single student.",
+    async () => {
+      // Arrange
+      const assessmentConsolidatedData =
+        createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+      assessmentConsolidatedData.studentDataApplicationPDPPDStatus = "yes";
+
+      // Act
+      const calculatedAssessment =
+        await executePartTimeAssessmentForProgramYear(
+          PROGRAM_YEAR,
+          assessmentConsolidatedData,
+        );
+
+      // Assert
+      expect(calculatedAssessment.variables.awardEligibilityCSGP).toBe(true);
+      expect(
+        calculatedAssessment.variables.finalFederalAwardNetCSGPAmount,
+      ).toBeGreaterThan(0);
+    },
+  );
+
+  it("Should determine CSGP as not eligible when application PD/PPD status is not 'yes'.", async () => {
+    // Arrange
+    const assessmentConsolidatedData =
+      createFakeConsolidatedPartTimeData(PROGRAM_YEAR);
+    assessmentConsolidatedData.studentDataApplicationPDPPDStatus =
+      "noIDoNotHaveADisability";
```

Comment:
Can we change the description to 
"Should determine CSGP as not eligible when application PD/PPD status is 'noIDoNotHaveADisability'." and create one more to another option different than "yes"?

Reply:
Please disregard the comment. Checking different than yes should be good enough 😉